include/flamegpu/runtime/detail/SharedBlock.h

```diff
@@ -19,8 +19,8 @@ struct SharedBlock {
 #if !defined(FLAMEGPU_SEATBELTS) || FLAMEGPU_SEATBELTS
     unsigned int curve_type_size[curve::Curve::MAX_VARIABLES];
     unsigned int curve_elements[curve::Curve::MAX_VARIABLES];
-    unsigned int curve_count[curve::Curve::MAX_VARIABLES];
 #endif
+    unsigned int curve_count[curve::Curve::MAX_VARIABLES];
```

Comment:
This feels like a bug that was fixed to do with seatbelts elsewhere? Not sure if this is the correct value, mid review. Should double check this works with and without seatbelts just incase.

(this is just a note mid-review so feel free to ignor for now, but doubt I'll get through all of this in one go).

Reply:
I think there was a bug related to this, that you couldn't build with seatbelts turned off. It was probably in fujitsu and has been fixed here (not enough context in 2 lines of code shown to check off-hand).